app/workers/articles/update_organic_page_views_worker.rb

```diff
@@ -0,0 +1,37 @@
+module Articles
+  class UpdateOrganicPageViewsWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :medium_priority,
+                    lock: :until_executing,
+                    on_conflict: :replace,
+                    retry: 10
+
+    GOOGLE_REFERRER = "https://www.google.com/".freeze
+
+    def perform(article_id)
+      article = Article.find(article_id)
+      google_page_views = article.page_views.where(referrer: GOOGLE_REFERRER)
+      update_params = {}
```

Comment:
The old code in `PageViewsController` performed 3 `update_column` calls. I decided to instead aggregate all updates into a hash and perform a single update at the end.

Reply:
There's an essential difference between the two approaches: `update_column()` does not trigger callbacks, skips validations and doesn't update the timestamps. `update()` does. 

Should it be replaced by [update_columns()](https://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-update_columns) to keep the same behavior but only issue one update?